### PR TITLE
Add practice CRFs

### PR DESCRIPTION
**Pull request by pbchase**
@tlstoffs, I add your Practice CRFs in this pull request. Your input is _especially_ valued here.

### CODE REVIEW

File: docs/guide_for_admins_and_devs.md [hunk 1]
```diff
@@ -105,7 +105,9 @@ The technical skills needed are in JavaScript and PHP. Front-end web developers
...
+Once you have hired a developer, you will need to train them. Even the best candidates are not likely to know anything about REDCap. Train every developer as if they are to become a REDCap Admin. Have them watch all of VUMC's [REDCap training videos](https://projectredcap.org/resources/videos/). Assign them a complicated project building task or a series of tasks that will require they use all the popular features of REDCap. At the University of Florida's CTS-IT, every REDCap Admin and every REDCap developer must build a REDCap project based on Taryn Stoff's [Practice CRFs for Training Admins](download/Practice_CRFs_for_Training_Admins.docx). This exercise uses most of the core REDCap features a REDCap admin should know.
```
Comment (tlstoffs) [inline docs/guide_for_admins_and_devs.md]: Stoffs's

Comment (tlstoffs) [inline docs/guide_for_admins_and_devs.md]:
```suggestion
Once you have hired a developer, you will need to train them. Even the best candidates are not likely to know anything about REDCap. Train every developer as if they are to become a REDCap Admin. Have them watch all of VUMC's [REDCap training videos](https://projectredcap.org/resources/videos/). Assign them a complicated project building task or a series of tasks that will require they use all the popular features of REDCap. At the University of Florida's CTS-IT, every REDCap Admin and every REDCap developer must build a REDCap project based on Taryn Stoffs' [Practice CRFs for Training Admins](download/Practice_CRFs_for_Training_Admins.docx). This exercise uses most of the core REDCap features a REDCap admin should know.
```

**Comment by pbchase** [comment]
@tlstoffs, I fixed your name in commit ddfb8f7.  Can you merge this?

**Comment by pbchase** [comment]
Fixes #10

**Pull request merged**